libparsec/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 
 mod addr;
 mod cancel;
+pub mod cli;
```

Comment:
We probably don't want the cli to be included in the lib :thinking:

Reply:
Why ?

Reply:
The lib is not an executable so the CLI function would not be used.

Reply:
This is not the `cli`, but more like `cli_utils`